install.sh

```diff
@@ -234,6 +234,40 @@ libffi_install ()
     popd
 }
 
+libsecp256k1_build()
+{
+    make clean
+    ./autogen.sh
+    ./configure \
+    --enable-module-recovery \
+    --disable-jni \
+    --prefix "${jm_root}" \
+    --enable-experimental \
+    --enable-module-ecdh \
+    --enable-benchmark=no
+    make
+    if ! make check; then
+        return 1
+    fi
+}
+
+libsecp256k1_install()
+{
+    secp256k1_lib_tar='0d9540b13ffcd7cd44cc361b8744b93d88aa76ba'
```

Comment:
This is a commit from Oct 11 2019. Any particular reason for using so old version?

Reply:
The reason for using this commit is it's the one currently used by coincurve.

Choosing a commit for libsecp, as you perhaps know, is very problematic as they do not tag or release. Seemed like the conservative choice in absence of more detailed investigations (we use nothing new, we actually don't use the ECDH API at the moment, though we might in future).